webdriver-spec.html

```diff
@@ -742,6 +742,9 @@
 
   <p>Otherwise, let <var>parameters</var> be null.</p>
 
+ <li><p>If a <a>navigation</a> is currently underway <a>wait for
+  navigation to complete</a>.
```

Comment:
Is "currently underway" a defined concept for navigation? I don't think it is, so I don't know what this section means except in a handwavy way.